### PR TITLE
Set baseline branch for build

### PR DESCRIPTION
**Pull request by wsbaser**

### CODE REVIEW

File: test/jest-e2e.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "moduleFileExtensions": ["js", "json", "ts"],
   "rootDir": "../",
   "testEnvironment": "node",
-  "testRegex": ".e2e-spec.ts$",
+  "testRegex": ".e2e.spec.ts$",
```
Comment (pashidlos) [inline test/jest-e2e.json:5]: any reason for renaming?

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
     "@nestjs/cli": "^8.2.3",
     "@nestjs/schematics": "^8.0.8",
     "@nestjs/testing": "^8.4.1",
-    "@prisma/cli": "2.12.1",
+    "@prisma/cli": "^2.12.1",
```
Comment (pashidlos) [inline package.json:61]: it's important to keep exact `2.12.1`

File: package.json [hunk 3]
```diff
@@ -14,10 +14,10 @@
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test": "jest --rootDir ./src",
```
Comment (pashidlos) [inline package.json:17]: is this necessary?

File: .github/workflows/workflow.yml [hunk 4]
```diff
@@ -4,51 +4,53 @@ name: CI
...
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (pashidlos) [inline .github/workflows/workflow.yml:13]: please, extract this into separate PR to make each PR targeted to single change

**Review by pashidlos — CHANGES_REQUESTED** [review]

File: src/test-variations/test-variations.service.ts [hunk 5]
```diff
@@ -80,20 +80,20 @@ export class TestVariationsService {
    * @returns
    */
   async find(
-    createTestRequestDto: BaselineDataDto & { projectId: string; sourceBranch?: string }
+    createTestRequestDto: BaselineDataDto & { projectId: string; baselineBranchName?: string }
```
Comment (pashidlos) [inline src/test-variations/test-variations.service.ts:83]: should work without renaming

File: test/jest-e2e.json (hunk 1 above)
Comment (wsbaser) [reply test/jest-e2e.json:5]: I renamed files because otherwise it is not possible to start them from Jest Test Explorer. It uses settings from package.json to find tests and it is "testRegex": ".spec.ts$". If test has file name .e2e-spec.ts then it does not match the pattern and test is not executed. For the same reason I had to change rootDir and added testTimeout property.

File: package.json (hunk 2 above)
Comment (wsbaser) [reply package.json]: done

File: package.json (hunk 3 above)
Comment (wsbaser) [reply package.json:17]: yes, it is necessary. otherwise it is not possible to run tests from Test Explorer

File: .github/workflows/workflow.yml (hunk 4 above)
Comment (wsbaser) [reply .github/workflows/workflow.yml:13]: done

File: src/test-variations/test-variations.service.ts (hunk 5 above)
Comment (wsbaser) [reply src/test-variations/test-variations.service.ts:83]: Yes, I renamed to prevent having two names for one thing, but it is not necessary. I could not find any usages of this field in the code so I decided that renaming should be safe. But we can recover original name to be 100% sure.

Comment (pashidlos) [reply src/test-variations/test-variations.service.ts:83]: it was used the same way, renaming is the only change here

Comment (wsbaser) [reply src/test-variations/test-variations.service.ts]: recovered the old name
